.ci/release.sh

```diff
@@ -7,11 +7,6 @@ set -e
 git config --global user.name "Gluon Bot"
 git config --global user.email "githubbot@gluonhq.com"
 
-# Update substrate to release version
-mvn versions:use-releases -Dincludes=com.gluonhq:substrate -DgenerateBackupPoms=false
-substrateReleaseVersion=$(xmlstarlet sel -t -v "//_:dependency[_:artifactId='substrate']/_:version" pom.xml)
-git commit pom.xml -m "Update Substrate version to $substrateReleaseVersion"
```

Comment:
So where does this change happen? Who takes care of updating substrate to release?

Reply:
The person who is doing the release. 

Since release can only happen via tagging the repository, this change needs to be done before the tag and therefore cannot be a part of the release pipeline.

Reply:
I understand that, but that means we go back to square one, where most of the times that step was not done. Now with the enforcer plugin, it will be a failure? But the tag will be already applied?

Reply:
IMO, if we want to automate this the only way would be to move away from tag based releases.

If you have any other idea, please share.